pilot/pkg/model/service.go

```diff
@@ -923,6 +923,43 @@ type AmbientIndexes interface {
 	WorkloadsForWaypoint(WaypointKey) []WorkloadInfo
 }
 
+// NoopAmbientIndexes provides an implementation of AmbientIndexes that always returns nil, to easily "skip" it.
```

Comment:
This is just for now right?

Reply:
Oh this has always existed; I just moved it during some refactoring